src/backend/distributed/executor/executor.c

```diff
@@ -404,83 +405,78 @@ static TaskExecutionState TaskExecutionStateMachine(ShardCommandExecution *
 
 
 /*
- * CitusExecScan is called when a tuple is pulled from a custom scan.
- * On the first call, it executes the distributed query and writes the
- * results to a tuple store. The postgres executor calls this function
- * repeatedly to read tuples from the tuple store.
+ * AdaptiveExecutorExecScan is called via CitusExecScan on the
+ * first call of CitusExecScan. The function fills the tupleStore
+ * of the input scanScate.
  */
 TupleTableSlot *
-CitusExecScan(CustomScanState *node)
+AdaptiveExecutorExecScan(CustomScanState *node)
```

Comment:
this should probably have a different name since its semantics are different from other ExecScan functions